test/prototype/mx_formats/test_mx_dtensor.py

```diff
@@ -25,6 +25,7 @@
 from tqdm import tqdm
 
 from torchao.prototype.mx_formats import MXLinearConfig
+from torchao.prototype.mx_formats.config import MXFP8CastKernelChoice
```

Comment:
sorry, `MXFP8Dim1CastKernelChoice`? since for dim0 we are always using torch.compile

Reply:
It was originally `MXFP8Dim1CastKernelChoice` but [here](https://github.com/pytorch/ao/pull/2554#discussion_r2210140775) we discussed naming it `MXFP8CastKernelChoice` for potentially adding support for dim0 and dim0+dim1 casts as well. I don't have strong opinions either way